RFC/src/RFC-0311_AssetTemplates.md

```diff
@@ -107,8 +107,8 @@ The instruction is in JSON format and MUST contain the following fields:
 | **template-specific data**         | Object        | Template-specific metadata can be defined in this section                                                                   |
 | **Signatures**                     |               |                                                                                                                             |
 | metadata_hash                      | `u256`        | A hash of the previous three sections' data, in canonical format (`m`)                                                      |
-| creator_sig                        | `u256`        | A digital signature of the message `H(R || P || m)` using the asset creator’s private key corresponding to the `issuer` PKH |
-| raid_signature                     | `u256`        | Digital signature for the given RAID, signing the message `H(R || P || RAID_ID || m)`                                       |
+| creator_sig                        | `u256`        | A digital signature of the message `H(R ‖ P ‖ m)` using the asset creator’s private key corresponding to the `issuer` PKH |
+| raid_signature                     | `u256`        | Digital signature for the given RAID, signing the message `H(R ‖ P ‖ RAID_ID ‖ m)`                                       |
```

Comment:
I think we are missing (a) the FQDN information and (b) RAID_ID signature as per RFC_0301 in the template. Picking up an asset's registration information, a VN should be able to verify all verifiable information contained within without requesting additional information, including the verifying the RAID_ID. 

If there is a field that says  `| fqdn   |  "No FQDN"|` the RAID_ID and its signature `s,R` can be fully verified using the `issuer` PKH.

In case of `|fqdn   |  "disney.com"|` the RAID_ID and its signature `s,R` can be fully verified by querying the OpenAlias TXT DNS record and by comparing the `issuer` PKH.

Reply:
This PR is only to fix the broken rendering, but to answer your question:
i) Yes, you're right. The FQDN must be included in the instruction. Nice catch.
ii) The public nonce for the signature needs to be provided too.
Then the validation step (which is done by VNs on receipt of this instruction) will
a) look up the RAID_ID from the FQDN given in the issuer field
b) Compare the RAID_ID to the one provided.
c) If they match, compare the public key to the public key (hash) provided. 
c) If they match, validate the signature against the Challenge built up from R, P, and m viz. 
  `sG ?= R + eP`

Reply:
Captured in #178